weblate/templates/translate.html

```diff
@@ -216,20 +216,23 @@ <h4 class="card-title">
                           {% for action, value, name in flag_actions %}
                             <li>
                               <a href=""
-                                 class="link-post"
+                                 class="dropdown-item link-post"
                                  data-href="{% url 'edit_context' pk=unit.pk %}"
                                  data-params='{"{{ action|escapejs }}": "{{ value|escapejs }}", "next": "{{ this_unit_url|escapejs }}"}'>{{ name }}</a>
                             </li>
                           {% endfor %}
                         {% endif %}
                         {% if user_can_add_unit %}
                           <li>
-                            <a href="#" data-bs-toggle="modal" data-bs-target="#add-unit-form">{% translate "Add variant of this string" %}</a>
+                            <a class="dropdown-item"
+                               href="#"
+                               data-bs-toggle="modal"
+                               data-bs-target="#add-unit-form">{% translate "Add variant of this string" %}</a>
                           </li>
                         {% endif %}
                         {% if user_can_translate and unit.translation.component.is_multivalue %}
                           <li>
-                            <a href="" class="add-alternative-post">{% translate "Add alternative translation" %}</a>
+                            <a class="dropdown-item" href="" class="add-alternative-post">{% translate "Add alternative translation" %}</a>
```

Comment:
H037 235:31 Duplicate attribute found. class

```suggestion
                            <a class="dropdown-item add-alternative-post" href="" >{% translate "Add alternative translation" %}</a>
```